maro/rl/learning/policy_manager.py

```diff
@@ -319,7 +341,9 @@ def policy_host(
         log_dir (str): Directory to store logs in. Defaults to the current working directory.
     """
     policy_dict = {}
-    proxy = Proxy(group, "policy_host", {"policy_manager": 1}, component_name=f"POLICY_HOST.{host_idx}", **proxy_kwargs)
+    proxy = Proxy(
```

Comment:
The whole policy_host function should remain as is. This is for distributed policy management where a single node does not have enough capacity to contain all policy instances. You should write a separate function, e.g., def grad_worker(), that plays the sole role of executing grad tasks. Each policy host should contain a fixed subset of policies and is the producer of grad tasks to be executed remotely by the grad workers.